wifiphisher/pywifiphisher.py

```diff
@@ -187,7 +187,8 @@ def shutdown(template=None, network_manager=None):
         template.remove_extra_files()
 
     print '[' + R + '!' + W + '] Closing'
-    sys.exit(0)
+    for c in phishinghttp.creds:
+        sys.exit('[' + R + '+' + W + '] Captured credentials:' + R + c)
```

Comment:
I saw the problem and pushed a fix yesterday. It should be good now. Thanks for reporting it.

Yes, a logger would be great. We can introduce some new features then:

1) A Verbose level. The logger method will output all the necessary information accordingly.

2) Output to stdout or a text file (or both) depending on user options.